boards/xtensa/intel_s1000_crb/doc/intel_s1000.rst

```diff
@@ -0,0 +1,248 @@
+to the network.[1]_
+
+The features include the following:
+
+- Digital Signal Processors
+
+  - Dual Tensilica LX6 cores @ 400 MHz with HiFi3 DSP
+  - Single precision scalar floating-point instructions
+  - 16KB 4-way I$; 48KB 4-way D$
+  - Up to 2400 DMIPS, 3.2 GMACS (16x16), 800 MFLOPS of Compute
+
+- Speech Accelerators
+
+  - A GMM (Gaussian Mixture Model) and neural network accelerator
+  - Low power keyboard and limited vocabulary recognition
+  - Up to 9.6 GMACS (16x16) of compute
+
+- Internal Memory
+
+  - 4MB shared embedded SRAM
+  - 64KB embedded SRAM for streaming samples in low power mode
+
+- External Memory Interfaces
+
+  - Up to 8MB external 16-bit PSRAM
+  - Up to 128MB external SPI flash
+
+- I/O Interfaces
+
+  - Host I/O: SPI for command and control, I2S for streaming audio, IRQ, reset, wake, optional USB 2.0 HS device
+  - Microphone: I2S/TDM 9.6 MHz max. bit clock
+  - Digital Microphone: 4 PDM ports 4.8 MHz max. bit clock
+  - Speaker: I2S/TDM 9.6 MHz max. bit clock
+  - Instrumentation: I2C master @ 100/400 MHz
+  - Debug: UART Tx/Rx/RTS/CTS up to 2.4 Mbaud/s
+  - GPIO: 10 mA sink/source, 8x PWM outputs
+
+- Power Management / Consumption
+
+  - Low power idle (memory retention); voice activity detection; play through; full active
+  - Clock and power gating support
+  - < 20 mW voice activity detection
+  - < 250 mW full active
+
+- Package (preliminary): FCCSP132 7.45 x 8.3mm 0.6/0.7mm pitch staggered/orthogonal
+
+- Temperature Range:- Commercial: 0 to 70 degree C; industrial: -40 to +85 degree C
+
+System requirements
+*******************
+
+Prerequisites
+=============
+
+The Xtensa 'toolchain'_ i.e. XCC is required to build this port. This needs a
+license and is available for Linux and Windows. Please contact Intel for getting
+the license.
+
+Set up build environment
+========================
+
+To install the Xplorer software, log on to https://xpg.cadence.com/login/gen/ten4genlogin.html.
+using your browser. Please contact Intel for the login name and password. Once you have entered
+the login name and password, select version as RF-2015.3 to download
+"Xplorer-6.0.3-linux-installer.bin".
+
+Run the installer using these commands:
+
+.. code-block:: console
+
+   cd ~/Downloads
+   chmod +x Xplorer-6.0.3-linux-installer.bin
+   sudo ./Xplorer-6.0.3-linux-installer.bin
+
+Xplorer software will be installed into the /opt/xtensa folder. Please note a dialogue box
+should pop-up after running this command. Otherwise, it means your system is missing some
+package which is preventing successful installation, most probably gtk2-i686.  You can
+install this missing package with:
+
+.. code-block:: console
+
+   sudo apt-get install gtk2-i686
+
+The Xtensa processor in the S1000 board (cavs21_LX6HiFi3_RF3_WB16) is not
+supported by the stock RF-2015.3 (Xplorer-6.0.3) tools installed above. Download
+the Cadence XCC build for cavs21_LX6HiFi3_RF3_WB16 (a .tgz file) by contacting
```

Comment:
again, how do you "contact Intel"?  How do you download the tgz file?  (no link is provided).